lib/network.py

```diff
@@ -578,7 +582,8 @@ def start_interface(self, server_key):
                 self.print_error("connecting to %s as new interface" % server_key)
                 self.set_status('connecting')
             self.connecting.add(server_key)
-            c = Connection(server_key, self.socket_queue, self.config.path)
+            c = Connection(server_key, self.socket_queue, self.config.path,
+                lambda x: x.bad_certificate.append_weak(self.on_bad_certificate))
```

Comment:
Oops.. Sorry.. 1 more style thing.. I believe this:

```python
    Connection(arg1, arg2, 
               arg3, arg4...)
```

Is preferred to this:

```python
    Connection(arg1, arg2, 
    arg3, arg4...)

```

As per some stupid PEP8 or something like that...

Reply:
Ideally we'd have some automatic checks for that using autopep8

Reply:
Relevant section: https://www.python.org/dev/peps/pep-0008/#indentation

Note: we don't even really follow pep8 in this codebase... but I personally just pick and choose what to follow from it...

Reply:
Ok I see you already fixed it, thanks.